norse/torch/module/izhikevich.py

```diff
@@ -0,0 +1,201 @@
+import torch
+
+from ..functional.izhikevich import (
+    IzhikevichState,
+    IzhikevichRecurrentState,
+    IzhikevichSpikingBehaviour,
+    izhikevich_step,
+    izhikevich_recurrent_step,
+)
+
+from norse.torch.module.snn import SNNCell, SNN, SNNRecurrent, SNNRecurrentCell
+
+
+def _convert_to_recurrent_state(state: IzhikevichState) -> IzhikevichRecurrentState:
+    z = torch.as_tensor(0)
+    rstate = IzhikevichRecurrentState(z=z, v=state.v, u=state.u)
+    return rstate
+
+
+class IzhikevichCell(SNNCell):
+    """Documentation WIP
```

Comment:
Just remove?

Reply:
I haven't finished writing the documentation of all the classes to match the documentation of the LIF neuron yet, hence why I left it